lib/client/chart.js

```diff
@@ -73,9 +73,15 @@ function init (client, d3, $) {
   var contextYDomain = [utils.scaleMgdl(36), utils.scaleMgdl(420)];
 
   function dynamicDomain() {
-    var mult = 1.3
+    // allow y-axis to extend all the way to the top of the basal area, but leave room to display highest value
+    var mult = 1.15
       , targetTop = client.settings.thresholds.bgTargetTop
-      , mgdlMax = d3.max(client.entries, function (d) { return d.mgdl; });
+      // filter to only use actual SGV's (not rawbg's) to set the view window.
+      // can switch to Logarithmic (non-dynamic) to see anything that doesn't fit in the dynamicDomain
+      , mgdlMax = d3.max(client.entries, function (d) { if ( d.type === 'sgv') { return d.mgdl; } });
```

Comment:
@scottleibrand : shouldn't this function always return a value, if you're finding the max, you can return `0` for non sgv values. Perhaps this fixes the unit tests, see travis or https://gist.github.com/PieterGit/5aada655fb268559bdcd229f9d621de1